ci/tests/20-fputil.bash

```diff
@@ -8,32 +8,63 @@ export SCRIPT_DIR="$(dirname ${BASH_SOURCE})/.."
 . "${SCRIPT_DIR}/helpers.bash"
```

Comment:
Can this file be deleted?  It seems to be unused with a preference for other files.

Reply:
This is mostly used for fail_and_stop() function. It just provides the errors from the log. If we wanted to move away from logging onto a separate file, we wouldn't need this file.

Reply:
I don't see where that function is defined in this file?  I am talking about deleting: `ci/tests/20-fputil.bash`

All I see is an unused loop and a bunch of commented-out code.

Reply:
Oh yes, I'm sorry. I think I misread the comment. I'll delete it.